packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewDetails.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+
+import { Card, Column, Text, InfoItem } from '@trezor/components';
+import { GeneralPrecomposedTransactionFinal } from '@suite-common/wallet-types';
+import { spacings } from '@trezor/theme';
+
+import { Translation } from 'src/components/suite';
+
+const Pre = styled.pre`
```

Comment:
Shouldn't we have component for this?

Reply:
I think it might be a typographyStyle instead. Then we can use it in Text component <Text as="pre">